Parsers/ASimRegistryEvent/Parsers/vimRegistryEventMicrosoftSysmon.yaml

```diff
@@ -12,7 +12,8 @@ References:
   Link: https://aka.ms/ASimRegistryEventDoc
 - Title: ASIM
   Link: https:/aka.ms/AboutASIM
-  Description: ASIM Sysmon Registry Creation Event Parser (event number 12, 13, 14) from "Event" and "WindowsEvent" tables.
+Description: |
+      This ASIM parser supports normalizing Microsoft Sysmon logs, produced by the Windows Forwarded Events connector, to the ASIM Registry Event normalized schema (event number 12, 13, 14) from "Event" and "WindowsEvent" tables.
```

Comment:
-> "This ASIM parser supports normalizing Microsoft Sysmon events (event number 12, 13, 14) from "Event" and "WindowsEvent" tables, to the ASIM Registry Event normalized schema .

I removed the connector name as Event is produced by another (yet to be identified) connector.